model/src/main/kotlin/config/ScopeExclude.kt

```diff
@@ -19,12 +19,14 @@
 
 package com.here.ort.model.config
 
+import com.fasterxml.jackson.annotation.JsonIgnore
+
 /**
  * Defines a scope that should be excluded.
  */
 data class ScopeExclude(
         /**
-         * The name of the scope.
+         * A regular expression to match the names of scopes to exclude.
```

Comment:
Maybe say "a regular expression string" or "string representing a regular expression" (or swap "string" with "pattern") to emphasize it's not a `Regex` object? Speaking of it, *could* it be a `Regex` directly as `Regex` has a constructor taking a `String`, and its `toString()` returns the pattern?

Reply:
Using `Regex` directly works, done.